xLights/SeqFileUtilities.cpp

```diff
@@ -2267,7 +2267,7 @@ void xLightsFrame::ImportSuperStar(const wxFileName& filename)
     for (size_t i = 0; i < _sequenceElements.GetElementCount(); i++) {
         if (auto p = _sequenceElements.GetElement(i))
             model = p->shared_from_this();
-        if (!model || model->GetType() == ElementType::ELEMENT_TYPE_MODEL) {
```

Comment:
That change is not equivalent to the existing code.   Why touch it?  Do you know how to test the SuperStar import?

Reply:
This should already be resolved in 5e93928